conf/citest/pipelines/pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+cache_dir = /scratch
```

Comment:
Not sure if something is stored, but without that every time `pytest` is run, it raises a warning due to the permissions of the default cache (somewhere inside `linuxbrew`, if I recall correctly).

Reply:
Ok, so I have found the following information:
> ... the internal cache plugin creates a `.pytest_cache` subdirectory in `rootdir` to store its cross-test run state.
which means that if you run `pytest` once and some tests fail, you can run it again with `--lf` and it will run only those tests that failed last time.
Regarding the `rootdir`, not sure how it will affect. Having a look at the source code, it doesn't seem to do much since we are passing a specific JSON file (or folder/test in the case of unitary tests). So once I can test everything, I will give it a try in both locations and report the results on Slack.